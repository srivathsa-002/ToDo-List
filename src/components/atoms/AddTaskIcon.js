import React from 'react';
import { Grid } from "@material-ui/core";
import AddIcon from "@material-ui/icons/Add";


export default class AddTaskIcon extends React.Component {
    render() {
        return (
            <Grid container>
                <Grid item>
                    <AddIcon data-testid="addTask"/>
                    {"   "}
                </Grid>

                <Grid item style={{ paddingTop: "1px", paddingLeft: "5px" }}>
                    <span style={{ paddingLeft: "5px" }}>Add task</span>
                </Grid>
            </Grid>
        );
    }
}