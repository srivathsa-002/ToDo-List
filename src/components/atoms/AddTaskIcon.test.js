import React from "react";
import {render,fireEvent} from "@testing-library/react";
import AddTaskIcon from "./AddTaskIcon";
// import { faItalic } from "@fortawesome/free-solid-svg-icons";
import {create} from "react-test-renderer";

describe("AddTaskIcon testing", () => {
    test("Matches the snapshot", () => {
      const input = create(<AddTaskIcon />);
      expect(input.toJSON()).toMatchSnapshot();
    });
});

it("renders correctly", () => {
    const{getByTestId} = render(<AddTaskIcon />)

    expect(getByTestId("addTask")).toBeTruthy()
    // expect(queryByPlaceholderText("Todo...")).toBeTruthy()
})