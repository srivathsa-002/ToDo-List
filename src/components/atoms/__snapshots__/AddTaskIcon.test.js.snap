// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AddTaskIcon testing Matches the snapshot 1`] = `
<div
  className="MuiGrid-root MuiGrid-container"
>
  <div
    className="MuiGrid-root MuiGrid-item"
  >
    <svg
      aria-hidden={true}
      className="MuiSvgIcon-root"
      data-testid="addTask"
      focusable="false"
      viewBox="0 0 24 24"
    >
      <path
        d="M19 13h-6v6h-2v-6H5v-2h6V5h2v6h6v2z"
      />
    </svg>
       
  </div>
  <div
    className="MuiGrid-root MuiGrid-item"
    style={
      Object {
        "paddingLeft": "5px",
        "paddingTop": "1px",
      }
    }
  >
    <span
      style={
        Object {
          "paddingLeft": "5px",
        }
      }
    >
      Add task
    </span>
  </div>
</div>
`;
